cli/tools/bundler.rs

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+use crate::colors;
+use crate::file_watcher;
+use crate::flags::Flags;
+use crate::fs_util;
+use crate::info;
+use crate::module_graph;
+use crate::program_state::ProgramState;
+
+use deno_core::error::AnyError;
+use deno_core::futures::future::FutureExt;
+use deno_core::ModuleSpecifier;
+use std::path::PathBuf;
+
+pub fn bundle_module_graph(
+  module_graph: module_graph::Graph,
+  flags: Flags,
+  debug: bool,
+) -> Result<String, AnyError> {
+  let (bundle, stats, maybe_ignored_options) =
+    module_graph.bundle(module_graph::BundleOptions {
+      debug,
+      maybe_config_path: flags.config_path,
+    })?;
+  match maybe_ignored_options {
+    Some(ignored_options) if flags.no_check => {
+      eprintln!("{}", ignored_options);
+    }
+    _ => {}
+  }
+  debug!("{}", stats);
+  Ok(bundle)
+}
+
+/// Returns a closure which returns the module graph of
+/// the given source file. The closure can be passed to file_watcher.
+pub fn get_module_resolver(
+  flags: &Flags,
+  source_file: &str,
+) -> Box<
+  dyn Fn() -> file_watcher::FileWatcherFuture<
+    file_watcher::ModuleResolutionResult<module_graph::Graph>,
+  >,
+> {
```

Comment:
```suggestion
pub fn get_module_resolver(
  flags: &Flags,
  source_file: &str,
) -> impl Fn() -> file_watcher::FileWatcherFuture<
  file_watcher::ModuleResolutionResult<module_graph::Graph>,
> {
```

It'd be nice to use "impl Trait" instead of trait object :-)

Reply:
Thank you! I didn't know `impl Trait`.